olm/prepare-community-operators-update.sh

```diff
@@ -108,38 +110,58 @@ do
-  echo
-  echo "   - Last package pre-release version of local package: ${lastPackagePreReleaseVersion}"
-  echo "   - Last package release version of cloned 'community-operators' repository: ${lastPublishedPackageVersion}"
-  if [[ "${lastPackagePreReleaseVersion}" == "${lastPublishedPackageVersion}" ]] && [[ "${FORCE}" == "" ]]; then
-    echo "#### ERROR ####"
-    echo "Release ${lastPackagePreReleaseVersion} already exists in the '${platformSubFolder}/eclipse-che' package !"
-    exit 1
-  fi
-
-  echo "     => will create release '${lastPackagePreReleaseVersion}' in the following package folder :'${folderToUpdate}'"
-
-  mkdir -p "${folderToUpdate}/${lastPackagePreReleaseVersion}"
-  sed \
-  -e "/^  replaces: ${packageName}.v.*/d" \
-  -e "/^  version: ${lastPackagePreReleaseVersion}/i\ \ replaces: eclipse-che.v${lastPublishedPackageVersion}" \
-  -e "s/${packageName}/eclipse-che/" \
-  "${packageBaseFolderPath}/${lastPackagePreReleaseVersion}/che-operator.clusterserviceversion.yaml" \
-  > "${folderToUpdate}/${lastPackagePreReleaseVersion}/eclipse-che.v${lastPackagePreReleaseVersion}.clusterserviceversion.yaml"
-
-  echo
-  echo "   - Update the CRD files"
-  cp "${packageBaseFolderPath}/${lastPackagePreReleaseVersion}/org_v1_che_crd.yaml" \
-  "${folderToUpdate}/${lastPackagePreReleaseVersion}/checlusters.org.eclipse.che.crd.yaml"
-  cp "${packageBaseFolderPath}/${lastPackagePreReleaseVersion}/org.eclipse.che_chebackupserverconfigurations_crd.yaml" "${folderToUpdate}/${lastPackagePreReleaseVersion}/org.eclipse.che_chebackupserverconfigurations_crd.yaml"
-  cp "${packageBaseFolderPath}/${lastPackagePreReleaseVersion}/org.eclipse.che_checlusterbackups_crd.yaml" "${folderToUpdate}/${lastPackagePreReleaseVersion}/org.eclipse.che_checlusterbackups_crd.yaml"
-  cp "${packageBaseFolderPath}/${lastPackagePreReleaseVersion}/org.eclipse.che_checlusterrestores_crd.yaml" "${folderToUpdate}/${lastPackagePreReleaseVersion}/org.eclipse.che_checlusterrestores_crd.yaml"
-  echo
-  echo "   - Update 'stable' channel with new release in the package descriptor: ${destinationPackageFilePath}"
-  sed -e "s/${lastPublishedPackageVersion}/${lastPackagePreReleaseVersion}/" "${destinationPackageFilePath}" > "${destinationPackageFilePath}.new"
-  mv "${destinationPackageFilePath}.new" "${destinationPackageFilePath}"
-  echo
+  for channel in $(echo $STABLE_CHANNELS | tr "," " ")
+  do
+    if [[ $channel == "stable-all-namespaces" && $platform == "kubernetes" ]];then
+      continue
+    fi
+    lastPackagePreReleaseVersion=$(yq -r '.channels[] | select(.name == "'$channel'") | .currentCSV' "${sourcePackageFilePath}" | sed -e "s/${packageName}.v//")
+    lastPublishedPackageVersion=$(yq -r '.channels[] | select(.name == "'$channel'") | .currentCSV' "${destinationPackageFilePath}" | sed -e "s/eclipse-che.v//") 
```

Comment:
`lastPreReleasePackageVersion`
`lastPublishedPackageVersion` 
?

Reply:
`lastPreReleasePackageVersion` = The release to be pushed to Operator hub
`lastPublishedPackageVersion` = Latest release published in community operators